gsa/src/web/utils/testing.js

```diff
@@ -52,6 +52,14 @@ afterEach(cleanup);
 const queryAllByName = (container, name) =>
   queryAllByAttribute('name', container, name);
 
+const queryByName = (container, name) => {
+  const els = queryAllByName(container, name);
```

Comment:
What does els stand for? Without the proper background knowledge this is not a descriptive variable name for me

Reply:
It uses the same name pattern as dom-testing-library:

queryAll... => return all found elements as array or empty array if no element is found
query... => return first found element or null if no element is found
get... => return first found elment or raise exception if no element is found
...ByTestId => search by data-test-id attribute
...ByName => search by name attribute

Reply:
Thanks for the comprehensive answer. My question was merely about the variable naming though ;)

Reply:
`els` is an abbreviation for elements. I've used the same variables as in dom-testing-library
https://github.com/kentcdodds/dom-testing-library/blob/master/src/queries.js#L225